### ISSUE TITLE
Need runoff mapping files for CESM-CISM's Antarctica grid

### ISSUE THREAD

**Issue by billsacks** [ESMCI/ccs_config_cesm#3]
We need runoff mapping files for CESM-CISM's Antarctica grid, replacing the placeholders here:

https://github.com/ESMCI/cime/blob/c89e85d1091b28b9ff4f6403168ca92fd66285ee/config/cesm/config_grids_common.xml#L138-L141

**Comment by mvertens** [comment]
@billsacks - thanks for pointing this out. I will get this into the next PR.

**Comment by billsacks** [comment]
@mvertens thanks for your offer, but I want to make sure you know what you're offering here. This was an outstanding issue in CIME that I just transferred to this repository. There currently are no runoff mapping files for Antarctica, so these need to be created following the process for creating runoff mapping files we have used in the past for glc -> ocn (described here: https://escomp.github.io/cism-docs/cism-in-cesm/versions/master/html/new-grids.html#generating-glc-ocn-mapping-files - but possibly out of date since it has been a few years since I or anyone has done this).
